justaddcl/yujinelson.com
# [CU-2x1wjey] Update CV to 2022 version

## Background
My CV on the website is still the outdated version used in 2021. This should be updated to reflect my role at Duffel as well as the other changes in my CV.

## Changes
- Added 2022 version of CV
- Updated the `ResumeLink` component to point to the updated CV file
- Removed all other old versions of the CV


## How to test
- Click all of the `resume` links across all the pages, header, and footer and confirm that they correctly link to the 2022 CV
